eng/pipelines/templates/jobs/ci.yml

```diff
@@ -140,15 +131,44 @@ jobs:
       image: $(LINUXVMIMAGE)
       os: linux
 
-    variables:
-      ComponentDetection.SourcePath: '$(Build.SourcesDirectory)/sdk/${{ parameters.ServiceDirectory }}'
+    steps:
+    - template: /eng/pipelines/templates/steps/download-package-artifacts.yml
+
+    - template: /eng/pipelines/templates/steps/resolve-package-targeting.yml
+      parameters:
+        BuildTargetingString: ${{ parameters.BuildTargetingString }}
+        PackagePropertiesFolder: $(Build.ArtifactStagingDirectory)/PackageInfo
+
+    - template: ../steps/build-extended-artifacts.yml
+      parameters:
+        ServiceDirectory: ${{ parameters.ServiceDirectory }}
+        BeforePublishSteps: ${{ parameters.BeforePublishSteps }}
+        BuildDocs: false
+        RunApiStubGen: true
+        TestPipeline: ${{ parameters.TestPipeline }}
+        Artifacts: ${{ parameters.Artifacts }}
+
+  - job: 'Build_Documentation'
+    displayName: Build Docs
+    dependsOn:
+      - 'Build_Linux'
```

Comment:
The yaml sequence tabbing is inconsistent in both new and existing yaml. We should follow this PR with another for formatting?